cli/tests/unit/flash_test.ts

```diff
@@ -986,7 +943,7 @@ Deno.test(
     const server = Deno.serve({
       handler: async (request) => {
         assertEquals(await request.text(), "");
-        assertEquals(request.headers.get("cookie"), "foo=bar, bar=foo");
+        assertEquals(request.headers.get("cookie"), "foo=bar; bar=foo");
```

Comment:
This test was actually wrong, now it's working properly

Reply:
Really? I think the previous behavior is correct - mhhh

Reply:
That appears to be the case, I found similar issue in Node and `undici` as well as examples on MDN use `;` to separate them.

Reply:
Cookies are definitely `;`-separated.